pkg/app/piped/executor/terraform/terraform.go

```diff
@@ -80,8 +80,8 @@ func findTerraform(ctx context.Context, version string, lp executor.LogPersister
 	return path, true
 }
 
-func findCloudProvider(in *executor.Input) (name string, cfg *config.CloudProviderTerraformConfig, found bool) {
-	name = in.Application.CloudProvider
+func findCloudProvider(in *executor.Input) (cfg *config.PlatformProviderTerraformConfig, found bool) {
```

Comment:
```suggestion
func findPlatformProvider(in *executor.Input) (cfg *config.PlatformProviderTerraformConfig, found bool) {
```

Reply:
This PR only adds PlatformProviders as configuration of piped.
Lets me address those suggestions in another PR since this changed the behavior of Piped and I want to address those things on one PR 🙏